src/ObjectUtils/CoffFile.cpp

```diff
@@ -173,6 +173,9 @@ std::optional<SymbolInfo> CoffFileImpl::CreateSymbolInfo(
   // in DeduceDebugSymbolMissingSizesAsDistanceFromNextSymbol.
   symbol_info.set_size(kUnknownSymbolSize);
 
+  // We currently only support hotpatchable functions in elf files.
+  symbol_info.set_is_hotpatchable(false);
```

Comment:
Nit: Could you create an issue on github that we want to support that at some point and link that here? Also I would have the comment as a todo. Same below.

Reply:
It's not really a todo. We would need general support for user space instrumentation under windows for this to make sense. As long as this doesn't happen this is rather a todon't. 

I have something like a design doc for the hotpatchable support. This contains some finding about msvc. I should probably dump this into a github issue.

Reply:
Sounds good to me. Thanks!